client/src/client.ts

```diff
@@ -490,7 +490,7 @@ function getServerOptions(ctx: vscode.ExtensionContext, debug: boolean): lsp.Nod
   // Node module for the language server
   const args = constructArgs(ctx, viewEngine, vscode.workspace.isTrusted);
   const prodBundle = ctx.asAbsolutePath('server');
-  const devBundle = ctx.asAbsolutePath(path.join('dist', 'server', 'src', 'server.js'));
+  const devBundle = ctx.asAbsolutePath(path.join('bazel-bin', 'server', 'src', 'server.js'));
```

Comment:
this one is surprising? needed for debugging or for some other reason?

Reply:
This is when running the extension how locally. We don't copy anything from `bazel-bin` to `dist` so this is definitely necessary.

Reply:
👍